### DISCUSSION TITLE
Deezer Downloads

### DISCUSSION THREAD

**Discussion by burnshroom** [RandomNinjaAtk/arr-scripts#110] [Q&A]
I recently signed up for a trial deezer account and added my arl to the extended.conf file.  Before adding this downloads were happening often, but missing some items.  Since I added the arl lidarr has not downloaded anything since.

And in the logs I see a line that says:  2023-11-10 01:57:22 :: Audio :: 2.23 :: DEEZER :: ARL Token: Configured

But then later I see these lines:  

2023-11-10 01:58:37 :: Audio :: 2.23 :: 1 :: missing :: 3 of 1000 :: Mariah Carey :: Oh Santa! :: Single :: Download Attempt number 1
Paste here your arl:
Aborted!
2023-11-10 01:58:38 :: Audio :: 2.23 :: 1 :: missing :: 3 of 1000 :: Mariah Carey :: Oh Santa! :: Single :: ERROR :: download failed, missing tracks...
2023-11-10 01:58:38 :: Audio :: 2.23 :: 1 :: missing :: 3 of 1000 :: Mariah Carey :: Oh Santa! :: Single :: Retry Download in 1 second fix errors...

I have waited it out hoping a download would happen eventually, but it is now at 10 days without a download.  

Anyone have any ideas or insights?  

I saw a similar post from back in July that said their issue was fixed but I am still getting this error.

I even generated a new ARL but still the same results.

Logs: https://github.com/RandomNinjaAtk/arr-scripts/files/13317251/Audio-2023_11_10_01_57_AM.txt

**Comment by RandomNinjaAtk** [comment]
Your ARL token is possibly incorrect, not the full key. Or the script cannot read the conf file to get the token because you accidentally changed the line endings to a windows format which breaks compatibility.

Suggestion, double check or even get a new ARL token and try again. Also make sure the files line endings are in UNIX format.
https://www.google.com/search?client=firefox-b-1-d&q=notepad%2B%2B+line+endings+unix

**Reply by burnshroom** [reply]
I attached images of the actual ARL (blurred our partially) as well as the extended.conf file showing the unix endings are valid.  

But still getting the error. 

After your reply, I deleted the existing .conf file and restarted the container and it pulled a fresh copy.  I then got a new ARL and inserted it and restarted again and it still gives the same error.  

![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/123109/6c698c65-9df5-43c8-9063-3ff414a0743f)

![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/123109/e0218b22-26d4-4aa1-accc-c4470f2278a7)

**Comment by burnshroom** [comment]
Closing as I did a new install and it seems to be now recognizing the arl fine now.  New discussion coming though...
